packages/react-strict-dom/babel.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+module.exports = function ({ types: t }) {
+  const packageName = 'react-strict-dom';
+  const packageRuntime = 'react-strict-dom/dist/runtime';
+  const findImportDeclaration = (body, sourceValue) =>
+    body.find(
+      (node) =>
+        node.type === 'ImportDeclaration' && node.source.value === sourceValue
+    );
+  const findHtmlSpecifier = (specifiers) =>
+    specifiers.find((specifier) => specifier.imported.name === 'html');
+
+  return {
+    visitor: {
+      ImportDeclaration(path) {
+        if (path.node.source.value === packageName) {
+          const htmlSpecifier = findHtmlSpecifier(path.node.specifiers);
```

Comment:
It's an edge-case, but a use may import `html` from `react-strict-dom` twice (with different aliases) within the same file.

In such an edge-case, this plugin would end up inserting duplicate imports for `defaultStyles`.

There is a second edge-case that the variable name `defaultStyles` may already be used within the file.

Both of these issues can be solved by using the `@babel/helper-module-imports` package to insert import statements instead.

Reply:
Oh yeah, I noticed this edge case but forgot to account for it. Thanks for the pointer re: `@babel/helper-module-imports`